frontend/src/modules/organization/pages/organization-common-page.vue

```diff
@@ -3,36 +3,50 @@
     <lf-filter
       ref="organizationFilter"
       v-model="filters"
-      :config="organizationFilters"
+      :config="{}"
```

Comment:
Why don't we have filters?
Feature requirements: https://linear.app/lfx/project/data-quality-organizations-common-view-5927172f7790/overview#heading-6b9418fd